pkg/crc/gpg/gpg.go

```diff
@@ -0,0 +1,42 @@
+package gpg
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+
+	"github.com/ProtonMail/go-crypto/openpgp"
+	"github.com/code-ready/crc/pkg/crc/constants"
+)
+
+func Verify(filePath, signatureFilePath string) (err error) {
+	signed, err := os.Open(filePath)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err1 := signed.Close(); err1 != nil {
+			err = err1
+		}
+	}()
+
+	signature, err := os.Open(signatureFilePath)
+	if err != nil {
+		return err
+	}
+	defer func() {
+		if err1 := signature.Close(); err1 != nil {
+			err = err1
```

Comment:
Hmm if `err` is already set, won't this hide the initial error? (same question for the other occurrence in this function)

Reply:
yes https://trstringer.com/golang-deferred-function-error-handling/ looks like we need multi error in this case to append the error.

Reply:
Since we are open file in reading mode so in defer I think we are OK to just use `f.Close()` without handling the error. if we open file in write mode then it is more appropriate to handle defer function error.